config/vrchat-whitelist.txt

```diff
@@ -0,0 +1,12 @@
+soundcloud.com
+drive.google.com
+facebook.com
+*.nicovideo.jp
+*.twitch.tv
+*.vimeo.com
+*.youku.com
+*.youtube.com
+youtu.be
+mixcloud.com
+*.vrcdn.live
+*.vrcdn.video
```

Comment:
As a special exception to the standard VRChat whitelist, streamable.com should be included. This will make the defaults in line with the domain whitelist feature being added for ProTV3.

Reply:
done